spec/abstractops.html

```diff
@@ -69,6 +69,16 @@ <h1>IsLeapYear ( _year_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-temporal-isvalidcalendarname" aoid="IsValidCalendarName">
+    <h1>IsValidCalendarName ( _calendar_ )</h1>
+    <p>
+      The IsValidCalendarName abstract operation verifies that the _calendar_ argument (which must be a String value) represents a valid Unicode Calendar Identifier type, according to BCP47.
+    </p>
+    <p>
+      The abstract operation returns true if _calendar_, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>, is equal to one of the Unicode Calendar Indentifier types, according to BCP47, converted to uppercase as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>. It returns false otherwise.
```

Comment:
"Unicode Calendar Indentifier types"

* "Identifier", not "Indentifier"
* should "types" be "values"?

Do we not want to support custom identifiers?

Reply:
Done. The route for supporting custom identifiers is to patch `Temporal.Calendar.from`, which uses an alternate parser anyway, right?